ipa-core/src/helpers/transport/in_memory/transport.rs

```diff
@@ -162,15 +162,13 @@ impl<I: TransportIdentity> Transport for Weak<InMemoryTransport<I>> {
     }
 
     fn peers(&self) -> impl Iterator<Item = I> {
-        let this = self.identity();
-        let all: Vec<I> = self
-            .upgrade()
+        self.upgrade()
             .unwrap()
             .connections
             .keys()
             .copied()
-            .collect();
-        all.into_iter().filter(move |&id| id != this)
```

Comment:
why are you removing this filter? We want to broadcast to all the peers (not including ourselves).

Reply:
Discussed offline: in the in-memory we don't create a connection to ourselves.